lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -411,43 +411,53 @@ protected function _getCreateTableSQL($table, array $columns, array $options = a
     }
 
     /**
+     * Returns the SQL for a list of indexes in the database.
+     *
+     * @param string $database
+     * @param string $table
+     *
+     * @return string
+     */
+    public function getListIndexesSQL(?string $database, ?string $table): string
+    {
+        if ($database !== NULL && $database !== '/') {
+            $databaseIdentifier = $this->normalizeIdentifier($database);
+            $quotedDatabaseIdentifier = $this->quoteStringLiteral($databaseIdentifier->getName());
+        }
+        else {
+            $quotedDatabaseIdentifier = "(SELECT SYS_CONTEXT('userenv', 'current_schema') FROM DUAL)";
+        }
+        $tableCondition = NULL;
+        if ($table !== NULL) {
+            $tableIdentifier = $this->normalizeIdentifier($table);
+            $quotedTableIdentifier = $this->quoteStringLiteral($tableIdentifier->getName());
+            $tableCondition = "AND ind_col.table_name = $quotedTableIdentifier";
+        }
+        return <<<SQL
```

Comment:
Not sure if it's enforced by the style guide, but it's preferable to have empty lines around control structures (`if`, `return`, etc).